styles.css

```diff
@@ -679,4 +703,98 @@
       opacity: 0;
     }
   }
+
+  #popUp {
```

Comment:
- [ ] Your design is not fully responsive at mobile widths. Kindly make it more responsive. 
- [ ] I noticed that at mobile width the design stops being responsive altogether and sticks to the left of the screen. You can see the blank white area on the right. 
Let's fix these responsiveness issues. Cheers 👍  

![Personal Portfolio (1)](https://user-images.githubusercontent.com/60618877/179871839-c64e6883-7cea-4a4e-9f9a-9609f0ae20da.gif)